binr/r2r/run.c

```diff
@@ -334,12 +367,21 @@ R_API void r2r_subprocess_wait(R2RSubprocess *proc) {
 			}
 		}
 		if (FD_ISSET (proc->killpipe[0], &rfds)) {
+			timedout = false;
 			child_dead = true;
 		}
+		if (timedout) {
+			break;
+		}
 	}
 	if (r < 0) {
 		perror ("select");
 	}
+	return child_dead;
+}
+
+R_API void r2r_subprocess_kill(R2RSubprocess *proc) {
+	kill (proc->pid, SIGKILL);
```

Comment:
#if __UNIX__

Reply:
or use r_sys_kill

Reply:
This is already in the `#else` branch of `#if __WINDOWS__` where tons of unix api is used so should be fine.